src/components/ExpensiPicker.js

```diff
@@ -36,26 +36,24 @@ class ExpensiPicker extends PureComponent {
     }
 
     render() {
-        const {
-            label, isDisabled, ...pickerProps
-        } = this.props;
+        const pickerProps = _.omit(this.props, _.keys(propTypes));
```

Comment:
NAB, could maybe opt to be more explicit here vs. depending `propTypes`? We are creating a dependency where changing the `propTypes` will have a side effect of altering the `pickerProps` and not sure if that will be obvious to anyone modifying this (also might make zero difference so 🤷).

Reply:
I feel like `propTypes` is meant to be the main source of truth for the prop types for a component so I personally feel like this pattern is fine. Open to other ideas too, but it feels odd/unnecessary to duplicate all the props in the `_.omit` as they're already declared in `propTypes`

Reply:
Ok, I will just leave you with the thought where someone does not remember to add something to `propTypes` and unintentionally passes that prop in `pickerProps`. Again, might not be a problem, but the way to prevent it before was clearer i.e. anything not de-structured would be passed and we are changing this to anything not included in `propTypes` will be passed - which just feels less obvious.